docs/develop/testnet/api.md

```diff
@@ -1,3 +1,3 @@
 # Testnet API
 
-We hosted our testnet API endpoints [here](https://api.evmos.dev) and it uses Swagger.
+We hosted our testnet API endpoints [here](https://api.evmos.dev) and it uses Swagger.
```

Comment:
Ditto, not sure if we need a whole extra page for this. Instead this content could move to an overview page => docs/develop/testnet

Reply:
Will address this in another PR since a ticket was created for this.